lib/cylc/tests/test_review.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python2
+
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import pytest
+
+from tempfile import mkdtemp
+
+from cylc.review import CylcReviewService
+
+@pytest.mark.parametrize(
+    'cylc_8, log_dir, expected',
+    [
+        pytest.param(
+            True, True, True,
+            id="True when flow.cylc and log dir"
+        ),
+        pytest.param(
+            True, False, True,
+            id="True when flow.cylc and no log dir"
+        ),
+        pytest.param(
+            False, True, True,
+            id="True when suite.rc and log dir"
+        ),
+        pytest.param(
+            False, False, False,
+            id="False when suite.rc and no log dir"
+        ),
+    ],
+)
+def test_is_cylc8(cylc_8, log_dir, expected):
+    """Check is_cylc8 returns"""
+    temp_workflow_dir = mkdtemp()
```

Comment:
Why not just tmp_path fixture?

Reply:
I had thought it was neater this way, since you have to cast tmp_path to a string, but I've changed it anyway.